packages/grid/src/mouse-handlers/GridLinkMouseHandler.ts

```diff
@@ -0,0 +1,267 @@
+/* eslint class-methods-use-this: "off" */
+
+import clamp from 'lodash.clamp';
+import { assertNotNull } from '@deephaven/utils';
+import { isEditableGridModel } from '../EditableGridModel';
+import { EventHandlerResult } from '../EventHandlerResult';
+import Grid from '../Grid';
+import GridMouseHandler, { GridMouseEvent } from '../GridMouseHandler';
+import GridRange from '../GridRange';
+import GridRenderer from '../GridRenderer';
+import GridUtils, { GridPoint } from '../GridUtils';
+
+export interface LinkInformation {
+  left: number;
+  top: number;
+  right: number;
+  bottom: number;
+  width: number;
+  height: number;
+  value: string;
+  href: string;
+}
+
+class GridLinkMouseHandler extends GridMouseHandler {
+  timeoutId?: ReturnType<typeof setTimeout>;
+
+  private isHold = false;
+
+  private isDown = false;
+
+  private static currentLinkBox?: LinkInformation;
+
+  /**
+   * Checks if mouse is currently over a link and returns its information if so
+   * @param gridPoint Mouse position information
+   * @param grid The grid
+   * @returns A LinkInformation object if it exists and null otherwise
+   */
+  static isHoveringLink(
```

Comment:
Naming nitpick - `is...` implies a boolean return value, can we rename this to something like `getHoveringLinkInfo`?